### PR TITLE
Fix: z clipping particles

### PR DESCRIPTION
**Pull request by xezon**
This change fixes all z clipping particles I could find.

## Original Demo Trap

https://user-images.githubusercontent.com/4720891/188271577-e08a981e-6f2f-4e9c-88a0-b1b5a7effd9b.mp4

## Patched Demo Trap

https://user-images.githubusercontent.com/4720891/188271586-23ecd778-e6fc-4dce-89a3-d21aa2b2f1ea.mp4

## Original EMP

https://user-images.githubusercontent.com/4720891/188271605-0d7bf426-6035-4f2a-a833-ce8e0f54d9d9.mp4

## Patched EMP

https://user-images.githubusercontent.com/4720891/188271615-8a1ea8cf-e099-4c41-b820-9a3571378fe5.mp4

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/FXList.ini [hunk 1]
```diff
@@ -2067,6 +2067,7 @@ FXList FX_GattlingExplosionOneFinal
   End
   ParticleSystem
     Name = BattleMasterTankExplosionShockwave
+    Offset = X:0 Y:0 Z:2 ; Patch104p @bugfix from Z:0.00 to fix z clipping on flat terrain
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:2070]: All other uses of `BattleMasterTankExplosionShockwave` are at Z 2.

File: Patch104pZH/GameFilesEdited/Data/INI/FXList.ini [hunk 2]
```diff
@@ -8373,7 +8378,7 @@ FXList Nuke_WeaponFX_NukeCannon
 
   ParticleSystem
     Name = NukeCannonRing
-    Offset = X:0.0 Y:0.0 Z:0.0
+    Offset = X:0.0 Y:0.0 Z:5.0 ; Patch104p @bugfix from Z:0.00 to fix z clipping on flat terrain
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/FXList.ini:8381]: All other uses of `NukeCannonRing` are at Z 5

**Comment by commy2** [comment]
Nice.

**Review by commy2 — APPROVED** [review]
All values look reasonable and are consistent with other effect lists.

**Pull request merged**